### ISSUE TITLE
Internet Explorer compatibility?

### ISSUE THREAD

**Issue by PhpSriptKiddie** [sdrdis/jquery.flowchart#40]
Thanks for a great plugin! :+1: 

Any news on the IE compatibility? Most thing seems to work, but I cant select operators.

Thanks,

**Comment by morganzysman** [comment]
Hi,

I can't seem to be able to link operators using IE, any news on that.

Regards

**Comment by PhpSriptKiddie** [comment]
Yes. Theres a CSS fix in another thread somewhere that you need to apply.

**Comment by morganzysman** [comment]
I'm having a few issues with the fix. It worked but I know have an extra endpoint in IE. Have you had this issue?

Regards,
<img width="1440" alt="screen shot 2017-05-12 at 13 51 39" src="https://cloud.githubusercontent.com/assets/5576762/26048659/5fc5984a-3958-11e7-8ff5-a0407daa183c.png">

Morgan

**Comment by kaymatrix** [comment]
@PhpSriptKiddie Could you please provide that CSS link
